src/arrays/referencing_array_and_its_elements/hard/q8/WhichOneIsNext.java

```diff
@@ -0,0 +1,37 @@
+package arrays.referencing_array_and_its_elements.hard.q8;
+/*
+Write a program that takes an array of ten integers in ascending order. Because we need these numbers to be consecutive
```

Comment:
Change question text to: 
`Write a program that takes an array of ten integers in ascending order, checks if they are consecutive numbers, and prints the lowest missing number (if there is one). For example, in an array with elements {2, 3, 5, 6, 7, 10}, the lowest missing number is 4. To do this, write a method called missingNo that takes an ascending integer array and returns the lowest missing integer. This method should be generalized to work with any array length. If no integer is missing, return -1. Print out the number returned by missingNo. `